tests/testthat/test-table_with_settings.R

```diff
@@ -110,20 +110,17 @@ testthat::test_that("type_download_srv_table: downloading different output types
 })
 
 testthat::test_that("type_download_srv_table: pagination, lpp to small", {
-  testthat::expect_error(
-    shiny::testServer(
-      teal.widgets:::type_download_srv_table,
-      args = list(id = "tws", table_reactive = table_r),
-      expr = {
-        for (down_type in c(".txt", ".pdf")) {
-          session$setInputs(`pagination_switch` = TRUE)
-          session$setInputs(`lpp` = 1)
-          session$setInputs(`file_format` = down_type)
-          testthat::expect_true(file.exists(output$data_download))
-        }
+  shiny::testServer(
+    teal.widgets:::type_download_srv_table,
+    args = list(id = "tws", table_reactive = table_r),
+    expr = {
+      for (down_type in c(".txt", ".png")) {
```

Comment:
Any reason why we're checking `.png` here when the ui doesn't offer `.png` option?
https://github.com/insightsengineering/teal.widgets/blob/main/R/table_with_settings.R#L144

Instead, should we replace `.png` with `.pdf`, which we do offer?

Reply:
whoops my bad
thanks for the spot!

Reply:
```suggestion
      for (down_type in c(".txt", ".pdf")) {
```